projects/packages/jetpack-mu-wpcom/src/features/paragraph-block-placeholder/paragraph-block-placeholder.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Overrides the paragraph block placeholder for sites with write intent.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+use Automattic\Jetpack\Jetpack_Mu_Wpcom;
+
+// Turn off the feature on ETK plugin.
+define( 'MU_WPCOM_PARAGRAPH_BLOCK', true );
+
+/**
+ * Enqueue block editor assets.
+ */
+function wpcom_enqueue_paragraph_block_placeholder_assets() {
+	$asset_file          = include Jetpack_Mu_Wpcom::BASE_DIR . 'build/paragraph-block-placeholder/paragraph-block-placeholder.asset.php';
+	$script_dependencies = $asset_file['dependencies'] ?? array();
+	$version             = $asset_file['version'] ?? filemtime( Jetpack_Mu_Wpcom::BASE_DIR . 'build/paragraph-block-placeholder/paragraph-block-placeholder.js' );
+
+	wp_enqueue_script(
+		'paragraph-block-placeholder-script',
+		plugins_url( 'build/paragraph-block-placeholder/paragraph-block-placeholder.js', Jetpack_Mu_Wpcom::BASE_FILE ),
+		$script_dependencies,
+		$version,
+		true
+	);
+
+	global $post;
+	if ( is_post_with_write_intent( $post ) ) {
```

Comment:
Revisited this one, and maybe we can return early without enqueuing the script if the intent is not "write"

Reply:
Ooohh, nice idea! This is done.